cider/exec/template/CodeGenerator.h

```diff
@@ -175,7 +175,7 @@ class CodeGenerator {
                           const SQLTypeInfo&,
                           const Analyzer::Expr*,
                           const CompilationOptions&);
-
+  // only be called for In expr.
   std::unique_ptr<CodegenColValues> codegenCmpFun(const SQLOps,
```

Comment:
It should be also used by `codegenBinOper`.

Reply:
actually we have two function have same name, one is used in CompareIR.cpp to generate binary cmp op, this one is used in InValuesIR.cpp